spec.toml

```diff
@@ -0,0 +1,18 @@
+name = "Rust Spec Team"
+description = "Meetings for the Rust language specification team."
+
+[meta]
+includes = [ "_timezones.toml" ]
+
+[[events]]
+uid = "b7af715576cc229aaf8c532ea89bb6ace1c91a65"
```

Comment:
Don't we normally use an epoch for this? It doesn't really matter I guess

Reply:
It's just 160 bits of randomness encoded as hexidecimal.  It's a universally unique identifier, which is what [RFC 5545]( https://datatracker.ietf.org/doc/html/rfc5545) requires for this.